lang/cs/Org.Apache.REEF.IO/PartitionedData/FileSystem/FileSystemInputPartition.cs

```diff
@@ -123,23 +131,23 @@ private ISet<string> Download()
         /// provided by the Serializer
         /// </summary>
         /// <returns></returns>
-        public T GetPartitionHandle()
+        public IEnumerable<T> GetPartitionHandle()
```

Comment:
My reasoning for using `IEnumerable` is simple - it gives more flexibility. One can always return an `IEnumerable` of one item. It also works well with the new `ISerializer`, which only returns an object of `T` from a single `Stream` on deserialization. This means that you can easily match a `Stream` to a single object of `T`.